example/src/App.tsx

```diff
@@ -1,35 +1,65 @@
 import * as React from 'react';
-import { StyleSheet, View, Text } from 'react-native';
+import { StyleSheet, View, Text, Button } from 'react-native';
 import * as PiwikProSdk from 'react-native-piwik-pro-sdk';
 
 export default function App() {
   const [result, setResult] = React.useState<{ type: string; error?: Error }>({
     type: 'Loading',
   });
+  const [initialized, setInitialized] = React.useState(false);
+
+  const sendTestEvents = React.useCallback(async () => {
```

Comment:
Would be good if this `sendTestEvents` would update the `result` state so that any possible errors are clearer to the user. Now errors just produce "Unhandled Promise rejection" yellow boxes, while leaving the status as "Success".